app/assets/javascripts/user-satisfaction-survey.js

```diff
@@ -67,12 +67,21 @@
     setSurveyUrl: function(href) {
       var $surveyLink = $('#take-survey');
       var surveyUrl = $('#user-satisfaction-survey-container').data('survey-url');
+      var surveyStarts = new Date(2016, 1, 26).getTime();
+      var surveyEnds = new Date(2016, 1, 27, 23, 59, 59).getTime();
+
+      if (userSatisfaction.currentDate() >= surveyStarts &&
+          userSatisfaction.currentDate() <= surveyEnds) {
+        surveyUrl = 'https://www.surveymonkey.co.uk/r/2MRDLTW';
+      }
```

Comment:
I wondered if we should move all the URLs into the JavaScript to keep them consistent. After all, I believe the banner is only ever displayed if JavaScript is available. Maybe @dsingleton or @fofr have an opinion?